content.md

```diff
@@ -732,10 +736,12 @@ must shuffle it afterwards.
 
 Each of the phases is described in full detail in the following sections.
 
+## **START OF TURN**
+
+
```

Comment:
Do we really need an empty section? I'm fine with having it only in the Turn Sequence section. Same thing for end of turn.

Reply:
I'd be fine either way. Just added it for clarity :)